### PR TITLE
Added PostgreSQL to backend/database by Parmesh Bhatt

### LINKED ISSUE #1132: [Add]  PostgreSQL to backend/database
### Description

PostgreSQL is an open-source relational database and is used as a database management system in many applications.
You have to add how this database is working and how to use it.

### Screenshots

PostgreSQL

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by Parmesh119**
Closes #1132 

## Changes proposed
I have added the PostgreSQL/database in the backend category.
Please review it and If it is fine, please merge it to the main branch.

### CODE REVIEW

File: database/backend/database.json [hunk 1]
```diff
@@ -1,4 +1,11 @@
 [
+  {
+    "name": "PostgreSQL",
+    "description": "PostgreSQL is a free, open-source relational database and it is handling structured data and widely used database management system. ",
```
Comment (CBID2) [inline database/backend/database.json:4]: [Proofreading] 
```suggestion
    "description": "PostgreSQL is a free, open-source relational database that handles structured data and a widely used database management system. ",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @Parmesh119! :) I pointed out a proofreading error in my inline comment. In the future, consider running your descriptions through [Grammarly](https://www.grammarly.com/grammar-check) before making a pull request. Other than that, you're off to a great start! :)

**Comment by Parmesh119** [comment]
> Hi @Parmesh119! :) I pointed out a proofreading error in my inline comment. In the future, consider running your descriptions through [Grammarly](https://www.grammarly.com/grammar-check) before making a pull request. Other than that, you're off to a great start! :)

Ok

**Review by rupali-codes — APPROVED** [review]

**Review by CBID2 — APPROVED** [review]
LGTM

**Pull request merged**